src/components/ControlDropdown/ControlDropdown.tsx

```diff
@@ -18,19 +17,26 @@ import { Icon } from '../Icon';
 import { H4 } from '../typography';
 import { ControlDropdownProps } from './ControlDropdown.types';
 import { ControlDropdownPlacements } from './ControlDropdown.enums';
+import ControlledDropdown from '../Dropdown/ControlledDropdown';
+import { Placements } from '../Dropdown/Dropdown.types';
 
 const CloseButton = styled(Button)`
   padding: 0;
   width: ${pxToRem(16)};
   height: ${pxToRem(16)};
 `;
 
+const placementAdapter: Record<typeof ControlDropdownPlacements, Placements> = {
+  bottom: 'bottom',
+  'bottom-left': 'bottom-start',
+  'bottom-right': 'bottom-end',
+};
+
 const ControlDropdown: React.FC<ControlDropdownProps> = ({
   children,
   title,
   parentRef,
   isOpen = false,
-  onOpen,
   onClose,
```

Comment:
I'm bit worried about removing the `onOpen` prop but it seems to be working. Please remove also a reference to this prop in https://github.com/securityscorecard/design-system/blob/alpha/src/components/Datatable/ControlsModule/ControlsModule.tsx#L221-L226

Reply:
To me it seem's weird that we have an `onOpen` prop, since is a controlled component, there's no user action that could trigger an `open` action from within the panel